var/spack/repos/builtin/packages/parallelmergetree/package.py

```diff
@@ -17,9 +17,9 @@ class Parallelmergetree(CMakePackage):
 
     maintainers = ['spetruzza']
 
-    version('1.1.0',
+    version('1.1.2',
             git='https://bitbucket.org/cedmav/parallelmergetree.git',
-            tag='v1.1.0',
+            tag='v1.1.2',
             submodules=True)
```

Comment:
Is there a reason to get rid of `1.1.0`?  Normally new versions are added and old retained.

Reply:
@tldahlgren I'm sorry for the delayed reply, v1.1.1 fixes an issue with v1.0.2 - it didn't compile on g++ 6 and up because of an implicit include. The problem is that I didn't tag v1.1.1 at the right branch, so after a proper merge I tagged v1.1.2 at the right place. I'll go ahead and just retain the existing versions and add the newer one.